tests/phpunit/BltProject/ToggleModulesTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Acquia\Blt\Tests\BltProject;
+
+use Acquia\Blt\Tests\BltProjectTestBase;
+
+/**
+ * Class ToggleModulesTest.
+ *
+ * Verifies that setup:toggle-modules behaves as expected.
+ */
+class ToggleModulesTest extends BltProjectTestBase {
+
+  /**
+   * Verifies the modules for a given environment were enabled as expected.
+   *
+   * In the event no environment is specified, this test will be skipped.
+   *
+   * @group blt-project
+   */
+  public function testModulesEnabled() {
+    if (defined('BLT_ENV')) {
```

Comment:
Unit tests really shouldn't have conditionals in them. Will this ever be undefined?